### PR TITLE
Recreated Kubesimplify README.md

### PR DESCRIPTION
**Pull request by Abhishekvrt**
Hey everyone!!

I recreated the README.md. these are some of the things I updated  --

1.  Updated code and conduct and add some alive documentation 
2. East readable and made it new contributors friendly
3. added YouTube, discord and twitter socials in README
4. formatted and made it more structured....

Open for suggestions !!!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,70 @@
 <h1 align="center"><a href="https://kubesimplify.github.io/website/">Kubesimplify Website </a></h1>
 <p align="center"><i>The source code for Kubesimplify.com</i></p>
+Kubesimplify's official website. This codebase is what runs on [Kubesimplify](https://kubesimplify.github.io/website/). 
 
-This repository contains the assets required to build the Kubesimplify website and documentation. We're glad that you want to contribute! 
+This repository contains the assets required to build the Kubesimplify website and documentation.
 
+## Contribute
+We're glad that you want to contribute! 
+A first step to contributing is to start with something you think you can add value in te Community. Start attending Kubesimplify meetings, join the [Discord channels](https://discord.gg/uFBC8MyeeU) and subscribe to the [official youtube Channel](https://www.youtube.com/c/saiyam911). Ask in community, We always Welcome new Contributors.
```
Comment (AvineshTripathi) [inline README.md:9]:
```suggestion
A first step to contributing is to start with something you think you can add value in the community. Start attending Kubesimplify meetings, join the [Discord channels](https://discord.gg/uFBC8MyeeU) and subscribe to the [official youtube Channel](https://www.youtube.com/c/saiyam911). Ask in community, We always Welcome new Contributors.
```

Comment (AvineshTripathi) [reply README.md]: @pavangudiwada are we suppose to add this para in the readme as of now or maybe can you rephrase this to something that we should say to community folks who are not the ambassadors

File: README.md [hunk 2]
```diff
@@ -1,34 +1,70 @@
...
 - [Code of Conduct](code-of-conduct.md)
 
-<h1>Contributing Guidelines: </h1>
+### How to file an issue 
+Not ready to contribute code, but see something that needs work? While the community encourages everyone to contribute code, it is also appreciated when someone reports an issue.
+Make sure to adhere to the prompted submission guidelines while opening an issue. 
+
+### How to find something to work on
+Kubesimplify is Fresh starting Community so there is a lot to do. Be in touch with [Discord Server](https://discord.gg/uFBC8MyeeU), Twitter and [official youtube Channel](https://www.youtube.com/c/saiyam911). There are a lot of ongoing projects and you can contribute to them.  
+### How to open a pull request
```
Comment (AvineshTripathi) [inline README.md:20]:
```suggestion

### How to open a pull request
```

File: README.md [hunk 3]
```diff
@@ -1,34 +1,70 @@
...
+We recommend open an issue before editing and Pulling request. If you don't know "How to open a Pull request" We refer you to [this resource](https://opensource.com/article/19/7/create-pull-request-github)
 
-Kubesimplfy's new website. This codebase is what runs on [Kubesimplify](https://kubesimplify.github.io/website/). 
 
-For new developers getting started, here are some instructions to follow:
-Fork the repo by clicking the fork button. This will clone the repo in your account, changes done here will be shown in the main repo. 
- 
+## Prerequisites
+To use this repository and to Run Kubesimplify's website on your computer, you need the installed dependencies locally:
```
Comment (AvineshTripathi) [inline README.md:25]:
```suggestion
To use this repository and to Run Kubesimplify's website on your local device, follow the below steps:
```

File: README.md [hunk 4]
```diff
@@ -1,34 +1,70 @@
...
+
+Before you start, install the dependencies. Clone the repository and navigate to the directory:
```
Comment (AvineshTripathi) [inline README.md:27]:
```suggestion
```

**Comment by Abhishekvrt** [comment]
Thank-you @AvineshTripathi

File: README.md [hunk 5]
```diff
@@ -1,34 +1,70 @@
...
     $ npm start
 ```
+
 And then open up your web browser and go to [localhost:3000](http://localhost:3000).
 
-## <h1>Code of conduct</h1>
-Participation in the Kubesimplify community is governed by the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
+## Find something to work on
+
+The first step to getting starting contributing to Kubesimplify is to find something to work on. Help is always welcome, and no contribution is too small!
```
Comment (AvineshTripathi) [inline README.md:48]:
```suggestion
The first step to getting started contributing to Kubesimplify is to find something to work on. Help is always welcome, and no contribution is too small!
```

File: README.md [hunk 6]
```diff
@@ -1,34 +1,70 @@
...
+Become a part of Kubesimplify community by Contribute. All types of contribution are allowed (code contribution, Docs contribution and community part too)
```
Comment (AvineshTripathi) [inline README.md:49]:
```suggestion
Become a part of the Kubesimplify community by contributing. All types of contributions are allowed (code contributions, docs contributions, and community contributions too).
```

File: README.md [hunk 7]
```diff
@@ -1,34 +1,70 @@
...
+Kubesimplify's official website. This codebase is what runs on [Kubesimplify](https://kubesimplify.github.io/website/). 
 
-This repository contains the assets required to build the Kubesimplify website and documentation. We're glad that you want to contribute! 
+This repository contains the assets required to build the Kubesimplify website and documentation.
 
+## Contribute
+We're glad that you want to contribute! 
+A first step to contributing is to start with something you think you can add value in te Community. Start attending Kubesimplify meetings, join the [Discord channels](https://discord.gg/uFBC8MyeeU) and subscribe to the [official youtube Channel](https://www.youtube.com/c/saiyam911). Ask in community, We always Welcome new Contributors.
+
+The Contributor Guide provides detailed instruction on how to get your ideas and bug fixes seen and accepted, including:
```
Comment (AvineshTripathi) [inline README.md:11]:
```suggestion
```

File: README.md [hunk 8]
```diff
@@ -1,34 +1,70 @@
...
 - [Code of Conduct](code-of-conduct.md)
```
Comment (AvineshTripathi) [inline README.md:12]:
```suggestion
```

File: README.md [hunk 9]
```diff
@@ -1,34 +1,70 @@
...
+
+Here are some things you can do today to get started contributing:
+
+1. Help improve the Kubesimplify documentation
+2. Clarify code, variables, or functions that can be renamed or commented on
+3. Write test coverage
+4. Help triage issues
+
+## Community Code of Conduct
+We follow Code of conduct here. Participation in the Kubesimplify community is governed by the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md)
```
Comment (AvineshTripathi) [inline README.md:59]: Should this be CNCF code of conduct @saiyam1814 @anubha-v-ardhan

File: README.md [hunk 10]
```diff
@@ -1,34 +1,70 @@
...
+
+### For new developers getting started, here are some instructions to follow:
+1. Follow code of conduct 
+2. Be respectful to every contributer, We all are working as a team.
+3. Always open to new ideas and implementation.
+
+## Membership
+We encourage all contributors to become members. We aim to grow an active, healthy community of contributors, reviewers, and code owners. Learn more about requirements and responsibilities of membership on our [Discord channels](https://discord.gg/uFBC8MyeeU).
```
Comment (AvineshTripathi) [inline README.md:67]: I am not sure if we need this

Comment (Abhishekvrt) [reply README.md:66]: On this moment, of course, we don't need this. But soon we will in long term

File: README.md (hunk 9 above)
Comment (Abhishekvrt) [reply README.md:58]: @AvineshTripathi originally It was added as CNCF code of conduct.

**Comment by AvineshTripathi** [comment]
Can you squash this commits @Abhishekvrt

**Comment by Abhishekvrt** [comment]
@AvineshTripathi I think I done that!! 
review this...

**Comment by verma-kunal** [comment]
Hey @Abhishekvrt , i think due to some issue while squashing, all the previous commits have also been added to this PR!
I would suggest you to make a fresh PR for your changes 

Sounds good? Let me know if you'll need any help :)
